### DISCUSSION TITLE
Cannot change checkbox in Sidebar

### DISCUSSION THREAD

**Discussion by tom-lp-at** [fomantic/Fomantic-UI#1420] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Bug Report
if i use a checkbox in a (left) sidebar, it´s shown with it´s initial state, but cannot clicked to change. i reduced the complete code to the smallest possible, but the behaviour is always the same 

## Steps to reproduce
```
<div class="ui checkbox">
	<input type="checkbox"   id="check" name="check" checked="checked">
	<label for="check"></label>
</div>
```

## Expected result
change the state of the checkbox

## Actual result
checkbox stays in it's initial state

## Version
2.8.4 (4. March 2020)

I try different ways and reduce the whole code. Buttons/Icons/img with onclick in the left sidebar are working and the events are fired, but not for the checkbox (i remove the onchange above).
i try it with onclick and the events fires up, but the checkbox didn´t change his state...
if the checkbox is outside of the sidebar (for example in a content-div) the checkbox change his state as expected....
(i try it without ```for="check"``` too, also without name...)

**Comment by lubber-de** [comment, marked as answer]
Initialize your checkboxes by `$('.ui.checkbox').checkbox()`
See http://jsfiddle.net/7oqxjy26/

**Comment by tom-lp-at** [comment]
oh my God - i am so sorry to make (so stupid) noise :(
Thank you to bring me back to the right way...
It's - in the start - irritating to "initialize" basic html things like checkboxes :)
kr Tom
